src/cirrus/management/deployment_pointer.py

```diff
@@ -18,6 +18,7 @@
 
 # core required vars.  Not exclusive.
 REQUIRED_VARS = {
+    "CIRRUS_DATA_BUCKET",
```

Comment:
This is another reason to not use catalog.json. 1) We don't have a reliable way to manage catalog.json, so plenty of deployments may not have it. 2) This will require terraform changes to ensure this is set in the parameters, so we might as well do something better.